great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -244,7 +244,7 @@ def __init__(  # noqa: PLR0913
         base_directory,
         filepath_template=None,
         filepath_prefix=None,
-        filepath_suffix=None,
+        filepath_suffix=".json",
```

Comment:
Manual testing with a file-backed context showed that we didn't create a JSON file by default. Feels like all file-backed stores should do this so let's update the default